ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/apiprovider/ApiProviderToolbar.java

```diff
@@ -19,8 +19,8 @@
     private static final class Element {
         public static final By ROOT = By.cssSelector(".pf-c-page__main-breadcrumb");
         public static final By OPERATIONS_DROPDOWN = By.cssSelector(".operations-dropdown");
-        public static final By GO_TO_OPERATION_LIST_BUTTON = By.xpath(".//button[normalize-space()='Back to Operation List']");
-        public static final By EDIT_OPENAPI_DEFINITION = By.xpath(".//a[normalize-space()='View/Edit API Definition']");
+        public static final By GO_TO_OPERATION_LIST_BUTTON = By.xpath("//button[normalize-space()='Back to Operation List']");
```

Comment:
Looks like you can't do that, since it's then used as 
```getRootElement().$(Element.GO_TO_OPERATION_LIST_BUTTON)```
and that fails with ```java.lang.IllegalArgumentException: XPath starting from / searches from root```